pkg/runtime/security/token.go

```diff
@@ -8,7 +8,8 @@ import (
 /* #nosec */
 const (
 	// APITokenEnvVar is the environment variable for the api token
-	APITokenEnvVar = "DAPR_API_TOKEN"
+	APITokenEnvVar    = "DAPR_API_TOKEN"
+	AppAPITokenEnvVar = "APP_API_TOKEN"
 	// APITokenHeader is header name for http/gRPC calls to hold the token
 	APITokenHeader = "dapr-api-token"
```

Comment:
Are we using the same for AppTokenHeader? Also, shouldn't this be configurable?

Reply:
The rational here is that Dapr has a defined (thus non-configurable) header for two way authentication: both for App to Dapr and Dapr to App.

Since the app's code need to account for Dapr anyway, I feel like it's OK to make this a well known, non-configurable value. This removes another configuration users should care about that, and that doesn't have much justification to be made configurable.

So the value `dapr-api-token` in the context of App to Dapr means the token used to authenticate to Dapr, and in Dapr to App it means this is the token from Dapr that the app can choose to use if it wants to.